features/scv/jceks-impl/src/main/java/org/opennms/features/scv/jceks/JCEKSSecureCredentialsVault.java

```diff
@@ -102,9 +102,12 @@ public JCEKSSecureCredentialsVault(String keystoreFile, String password, boolean
         m_salt = Objects.requireNonNull(salt);
         m_iterationCount = iterationCount;
         m_keyLength = keyLength;
-        m_keystoreFile = new File(keystoreFile);
+        m_keystoreFile = new File(getKeyStoreFileName(keystoreFile));
```

Comment:
JCEKSSecureCredentialsVault is also loaded from [here](https://github.com/OpenNMS/opennms/pull/7717/files#diff-252bfeaa65a9c5e0ca977ad31367bef7f31bfbf50906d57ec41282f24da7c9b1R285). So need to take care of this case.

Reply:
Yes, defaultScv() also needs to be adjusted. OpenNMS is a software that has grown over decades. There are different parts of the code that use different technologies. Therefore, unfortunately, you have to consider as many different cases as you can instantiate a SCV. Since SCV in used as one basic scope in our meta-data DSL it is used in many places in OpenNMS itself and even in utilities that are running outside of OpenNMS but are part of our installation.